### PR TITLE
LPS-201084 Order of asset tag selector is not working fine

### PR DESCRIPTION
**Pull request by Stephy6**
https://liferay.atlassian.net/browse/LPS-201084

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/asset/asset-tags-item-selector-web/src/main/java/com/liferay/asset/tags/item/selector/web/internal/display/context/AssetTagsDisplayContext.java [hunk 1]
```diff
@@ -55,12 +55,12 @@ public SearchContainer<AssetTag> getTagSearchContainer() {
 
 		tagsSearchContainer.setOrderByCol("name");
 
-		boolean orderByAsc = false;
+		boolean orderByAsc = true;
```
Comment (ealonso) [inline modules/apps/asset/asset-tags-item-selector-web/src/main/java/com/liferay/asset/tags/item/selector/web/internal/display/context/AssetTagsDisplayContext.java:58]: @Stephy6 I think it is not necessary, we should have the same logic that we have in com.liferay.asset.tags.admin.web.internal.display.context.AssetTagsDisplayContext

File: modules/apps/asset/asset-tags-item-selector-web/src/main/java/com/liferay/asset/tags/item/selector/web/internal/display/context/AssetTagsDisplayContext.java [hunk 2]
```diff
@@ -123,7 +123,8 @@ private String _getOrderByType() {
 			return _orderByType;
 		}
 
-		_orderByType = ParamUtil.getString(_httpServletRequest, "asc");
+		_orderByType = ParamUtil.getString(
```
Comment (ealonso) [inline modules/apps/asset/asset-tags-item-selector-web/src/main/java/com/liferay/asset/tags/item/selector/web/internal/display/context/AssetTagsDisplayContext.java:126]: @Stephy6 We should keep the same behavior that we have in so the default value should be "asc"

**Comment by ruben-pulido** [comment]
ci:forward

**Comment by liulukiwi** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/143770

**Comment by liulukiwi** [comment]
ci:close

**Pull request closed without merging**